### DISCUSSION TITLE
New Metaverse Project

### DISCUSSION THREAD

**Discussion by shnkrcy** [IBAX-io/go-ibax#660] [Ideas]
My dream project is a metaverse universe, but in the others, the islands will not be empty. It will be prepared in advance as a company and the islands will consist of places where legendary movies and TV series take place. e.g; like hogwarts in harry potter, rohan minastirith isengaard in lord of the rings, avatar, dune.  People will not be able to change the basic things in this world, they will take the vacant place in Isengaard and build a house or open a commercial shop, various individual or collective games and competitions will be held.

Hayalimdeki proje bir metaverse evreni fakat diğerlerinde ayrılan noktası adalar boş olmucak. Firma olarak önceden hazırlanmış olucak ve adalar efsanevi film ve dizilerin geçtiği yerleden oluşucak. örneğin; harry potter'daki hogwarts, yüzüklerin efendisindeki rohan minastirith isengaard, avatar, dune gibi. İnsanlar bu dünyadaki temel şeyleri değiştiremicek isengaard'daki boş yeri alıp ev yapıcak veya ticari bi dükkan açıcak, çeşitli bireysel veya toplu oyunlar yarışmalar düzenlenicek

**Comment by hawkins09** [comment]
hahaha böyle bişey olsa gerçekten çok güzel olurdu, gerçekçilikle beraber orada yaşıyomuş hissi harika olurdu.
